core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/credentials/parameters/gcp/JsonParameters.java

```diff
@@ -0,0 +1,48 @@
+package com.sequenceiq.cloudbreak.api.endpoint.v4.credentials.parameters.gcp;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.credentials.parameters.Mappable;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class JsonParameters implements Mappable {
+
+    @ApiModelProperty(required = true)
+    private String credentialJson;
```

Comment:
is this contains a json?

Reply:
yes

Reply:
We have a Base64Deserializer and Base64Serializer class, I think it is usable here